dist/timetable.js

```diff
@@ -33,16 +33,28 @@ const formatTimeTable = (timeTable) => {
         id: parseInt(tr.td[5].span[0].a[0].$.rel.split('=')[1]),
         className: tr.td[1].span[0].a[0]._,
         time: tr.td[0].span[0]._,
-        canBook: !(tr.td[6] === 'Full' || tr.td[6] === 'Past')
+        location: clubLocation,
+        canBook: !(tr.td[7] === 'Add To Waiting List' || tr.td[7] === 'Past')
       });
     }
-
     return acc;
 
   }, {});
 };
 
+const combineTimeTables = (timeTables) => {
+  return new Promise((res, reject) => {
```

Comment:
You don't need a Promise here, you can just return the result of `timeTables.reduce(...)`, you can also use [ES6 arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) with implicit return.

Reply:
Also if you wanted to wrap the result into a Promise, you would probably need to do `Promise.resolve(timeTables.reduce(...))` but in this case you can just remove the Promise bit